libshaderc/README.md

```diff
@@ -0,0 +1,43 @@
+# libshaderc
+
+A library for compiling shader strings into SPIR-V.
+
+## Build Artifacts
+There are two main build artifacts that are created from a CMake build of
```

Comment:
Consider changing to `There are two main libraries generated after compilation.`?